app/Http/Controllers/Backend/MovieController.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use App\Models\Movie;
+use App\Models\Tag;
+use App\Models\Type;
+use App\Models\Video;
+use App\Models\Source;
+use App\Models\Channel;
+use App\Models\Playlist;
+use App\Models\Option;
+use App\Models\OptionValue;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Session;
+use Yajra\Datatables\Datatables;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Str;
+use App\Http\Requests\MovieRequest;
+
+class MovieController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+
+    public function index()
+    {
+        return view('backend.movies.index');
+    }
+
+    public function getData(Request $request)
+    {
+        $movies = Movie::select('id', 'name', 'nominations', 'slug', 'created_at');
+
+        if ($request->has('sort')) {
+            if ($request->get('sort') == 'Name') {
```

Comment:
```suggestion
            if ($request->get('sort') == 'name') {
```
sửa những chỗ tương tự